src/util/mod.rs

```diff
@@ -14,10 +15,29 @@ pub(crate) fn not_happening() -> Instant {
     *FUTURE
 }
 
+// The goal here is to make a constant "beginning of time" in both Instant and SystemTime
+// that we can use as relative values for the rest of str0m.
+// This is indeed a bit dodgy, but we want str0m's internal idea of time to be completely
+// driven from the external API using `Instant`. What works against us is that Instant can't
+// represent things like UNIX EPOCH (but SystemTime can).
+const HOURS_1: Duration = Duration::from_secs(60);
+static PAST: Lazy<(Instant, SystemTime)> = Lazy::new(|| {
+    // These two should be "frozen" the same instant. Hopefully they are not differing too much.
+    let now = Instant::now();
+    let now_sys = SystemTime::now();
+
+    let beginning_of_time = now.checked_sub(HOURS_1).unwrap();
+    // This might be less than 1 hour if the machine uptime is less.
```

Comment:
I don't think this comment is  true. If the machine is one of the architectures that cannot represent `Instant`s before the machine's boot time the line above would've paniced

Reply:
Right. Should be saturating sub.